REGION_FORK_RECOMMENDATIONS.md

```diff
@@ -0,0 +1,193 @@
+# Region Specific Fork Recommendations
+
+This document describes recommendations for keeping forks of [covidgreen/covid-green-backend-api](https://github.com/covidgreen/covid-green-backend-api) clean while also allowing for region specific changes. For example, the New York State fork [project-vagabond/covid-green-backend-api](https://github.com/covidgreen/covid-green-backend-api) contains CICD workflows specific to NYS which should not be contributed back to Covid Green. These recommendations adhere to the project's [Contributing Guidelines](CONTRIBUTING.md).
```

Comment:
Style nit: I'd generally like to see us wrap at <= 80 chars so that minor changes in text don't show up as entire paragraphs being changed. It's minor and not blocking for this PR tho

Reply:
Do you mean an explicit newline at or around 80 chars? When it comes to prose I tend to only newline between paragraphs so wrapping can be up to the viewer and the size of their monitor. That said, I don't mind adding explicit newlines.

Reply:
Yes, that's what I mean. For GitHub's markdown rendering, throwing in the newline won't throw off the wrapping (though it does limit when someone is looking at the raw text). As I said, just a style nit.

Reply:
Works for me. Will add.